### PR TITLE
Modify and update the comments

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

**What this PR does / why we need it**:
Modify and update the comments

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go [hunk 1]
```diff
@@ -117,8 +117,8 @@ type ClusterConfiguration struct {
 
 	// FeatureGates enabled by the user.
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
-
-	// The cluster name
+	
+	// ClusterName is the name for the cluster in kubeconfig.
```
Comment (stewart-yu) [inline cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go]: keep original

**Comment by xichengliudui** [comment]
@stewart-yu  Thank you for your prompt, I have revised it.

File: cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go [hunk 2]
```diff
@@ -118,7 +118,7 @@ type ClusterConfiguration struct {
 	// FeatureGates enabled by the user.
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
 
-	// The cluster name
+	// The cluster name	
 	ClusterName string `json:"clusterName,omitempty"`
```
Comment (stewart-yu) [inline cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go]: there are fmt error, please fix

Comment (xichengliudui) [reply cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go]: Thank you for your prompt, I have revised it.

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**